javadoc.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+echo "Javadoc-testing commit $TRAVIS_COMMIT"
+if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ -n "${GITHUB_TOKEN:-}" ]; then
```

Comment:
Because then we would report the check two times. A pull request always has the push build.